src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -8,9 +8,60 @@ import { metricError } from "config/metric-names";
 import { AppInstallation, FailedAppInstallation } from "config/interfaces";
 import { createAppClient } from "~/src/util/get-github-client-config";
 import { booleanFlag, BooleanFlags } from "config/feature-flags";
-import { isGitHubCloudApp } from "~/src/util/jira-utils";
+import { GitHubServerApp } from "models/github-server-app";
 import { getCloudOrServerFromGitHubAppId } from "utils/get-cloud-or-server";
 
+interface FailedConnection {
+	id: number;
+	deleted: boolean;
+	orgName?: string;
+}
+
+export interface InstallationResults {
+	fulfilled: AppInstallation[];
+	rejected: FailedAppInstallation[];
+	total: number;
+}
+
+interface SuccessfulConnection extends AppInstallation {
+	isGlobalInstall: boolean;
+}
+
+interface ConnectionsAndInstallations {
+	successfulConnections: SuccessfulConnection[]
+	failedConnections: FailedConnection[]
+	installations: InstallationResults
+}
+
+interface GitHubCloudObj {
+	successfulConnections: SuccessfulConnection[]
+	failedConnections: FailedConnection[]
+}
+
+interface GitHubServerObj {
+	gitHubBaseUrl:string
+	applications: ConnectionsAndInstallations[]
+}
+
+interface ViewConfigurationForGHCloud {
+	host: string
+	successfulConnections: SuccessfulConnection[]
+	failedConnections: FailedConnection[]
+	hasConnections: boolean
+	APP_URL?: string
+	csrfToken: string
+	nonce: string
+}
+interface ViewConfigurationForGHE {
+	host: string
+	gheServers: GitHubServerObj[]
+	ghCloud: GitHubCloudObj
+	hasConnections: boolean
+	APP_URL?: string
+	csrfToken: string
+	nonce: string
+}
```

Comment:
there's similarities between these 2 that should be it's own interface

Reply:
Interface corrected